ruby/game/game.rb

```diff
@@ -0,0 +1,234 @@
+# ----- PSEUDOCODE
+# Input word
+  # Loop until guess_count > word.length + 3
+    # Display word as spaces + guessed char (guessed_word)
+    # Input guess (char)
+    # Compare guess to word
+    # Check if char already guessed
+        # If so, prompt again
+    # Increment guess_count by 1
+    # Loop through array of word char's
+      # If guess = char in word, modify guessed_word at index
+        # Print correct guess
+      # Else
+        # Print incorrect guess
+    # Break if no '_' left in guessed_word array
+      # Declare 
+# Print either win or lose
+  # Based on won_game boolean
+
+class Game
+  attr_accessor :word, :guess_count, :guess, :guessed_word, :guessed_letters, :is_over, :won_game, :already_guessed
```

Comment:
Careful with attr_ methods.  Making these all accessible means that you can alter what they are outside of the class.

Are you doing this?
Should be changing them outside of the class?
What happens if you accidentally change `@word` into a string instead of an array?

Make sure you are protecting your attributes.